cihp13/cihp13_metadata_BARRA2.yml

```diff
@@ -0,0 +1,91 @@
+# Apply file attributes defined for the QDC scaled CMIP6 data which feeds into CIHP-13.
+#
+# This is for BARRA2 input.
+# metadata that would be nice to add but don't fit this YAML format:
+#  - global:
+#    - date_created

```

Comment:
@malcolmking At the moment the code automatically creates a global attribute called `creation_date` - is that ok or should it be `date_created`?

Reply:
`creation_date` is OK for CF compliance, but for ACDD compliance `date_created` is preferred.